src/header-components/header-component.js

```diff
@@ -9,7 +9,7 @@ function Header(props) {
         <h1>Rotten Tomatillos</h1>
       </h1>
       <HeaderSearch onChange={props.onChange}/>
-      <HeaderFilter/>
+      <HeaderFilter {...props} onFilter={props.onFilter}/>
```

Comment:
I like the usage of dot notation rather than calling the function on the element itself!